enclave_wrangler/main.py

```diff
@@ -142,6 +147,8 @@ def run(input_csv_folder_path):
 
 
     # --- Steph: check the failed cases 2/15/22-----------------------------
+    # @Steph: Can we remove this block yet? I don't mind keeping for a while for convenienice, but I like the practice
```

Comment:
Usually what I do if I want to save my test code is, I add a folder to `.gitignore`. I often call mine `_archive/`. And inside of that, I put personal notes, test files, etc, relating to the project; things I don't want to commit. In cases where I have some debugging / test code that I want to keep, I put a copy of it in there.